genai-on-vertex-ai/vertex_evaluation_services/gemini-curate-evaluation-data/curate_new_evals.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    csv_filename = "" # @param
</pre><p>is there also a sample CSV that can be provided to run this notebook ? You can use cell 14 in this notebook as reference to craft a short CSV file that users can run through - <a href="https://github.com/GoogleCloudPlatform/applied-ai-engineering-samples/blob/main/genai-on-vertex-ai/vertex_ai_extensions/notebooks/data_science_code_interpreter.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/applied-ai-engineering-samples/blob/main/genai-on-vertex-ai/vertex_ai_extensions/notebooks/data_science_code_interpreter.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>